package.json

```diff
@@ -18,7 +18,7 @@
   },
   "license": "BSD-3-Clause",
   "peerDependencies": {
-    "react": "^0.12.0"
+    "react": "^0.13.0"
```

Comment:
Let's keep 0.12 in there (like I said before) - we haven't done anything here that requires the use of 0.13 so we should be compatible with both versions.